processor/onboarding/_meta/fields.yml

```diff
@@ -0,0 +1,5 @@
+- key: apm-onboarding
```

Comment:
I assume you added this because a processor requires it. See my comment below, I think we should not make it a processor so this would not be required. But I see the point that potentially we want to have a special field like `up` or something like that that indicates it's available.